_layouts/default.html

```diff
@@ -94,7 +94,7 @@ <h1 class="in-logo">
           <div class="col-1-of-2">
             <div class="in-hire-us">
               <div>
-                <a href="http://codemancers.com/hire_us/index.html" target="_blank" class="in-btn">Hire us</a>
+                <a href="https://codemancers.com/contact/index.html" target="_blank" class="in-btn">Hire us</a>
```

Comment:
For out https,`www` is necessary. otherwise not redirecting. correct link will be `https://www.codemancers.com/contact/index.html`